src/app/search/page.js

```diff
@@ -57,35 +58,46 @@ export default async function Page({ searchParams }) {
             </h2>
             <div className="overflow-hidden rounded-lg bg-white shadow">
               <div className="p-6">
-                <Items
-                  data={users.map((user) => ({
-                    id: user.username,
-                    image: user.image,
-                    url: `/${user.username}`,
-                    urlText: user.name,
-                    description: user.bio,
-                    socials: user.platforms.map((platform) => ({
-                      icon: platform.name,
-                    })),
-                    meta: [
-                      // biggest reach
-                      {
-                        icon: WifiIcon,
-                        text: `Reach ${
-                          REACH().data[
-                            REACH().sortByLargest(user.platforms)[0].reach
-                          ].group
-                        }`,
-                      },
-                      // lowest price
-                      {
-                        icon: ArrowUpIcon,
-                        text: `From $${user.platforms[0].price}`,
-                      },
-                    ],
-                    tags: user.tags?.split(","),
-                  }))}
-                />
+                {users.length > 0 ? (
+                  <Items
+                    data={users.map((user) => ({
+                      id: user.username,
+                      image: user.image,
+                      url: `/${user.username}`,
+                      urlText: user.name,
+                      description: user.bio,
+                      socials: user.platforms.map((platform) => ({
+                        icon: platform.name,
+                      })),
+                      meta: [
+                        // biggest reach
+                        {
+                          icon: WifiIcon,
+                          text: `Reach ${
+                            REACH().data[
+                              REACH().sortByLargest(user.platforms)[0].reach
+                            ].group
+                          }`,
+                        },
+                        // lowest price
+                        {
+                          icon: ArrowUpIcon,
+                          text: `From $${user.platforms[0].price}`,
+                        },
+                      ],
+                      tags: user.tags?.split(","),
+                    }))}
+                  />
+                ) : (
+                  <Alert
+                    type="info"
+                    message="No results found"
+                    details={[
+                      "Try adjusting your search parameters.",
+                      "Check for typos or broaden your search criteria.",
```

Comment:
There are no ways for the user to have types as they are all drop downs

```suggestion
                      "Try to broaden your search criteria.",
```

Reply:
Changes look better. BTW for the future you can press accept on the suggestion ☝️ instead of writing it again

Reply:
Okay, thank you so much  for letting me know.
I will keep it in my mind .